services/leafSpaceIntegration/contactreservation/schemas.go

```diff
@@ -0,0 +1,44 @@
+package contactreservation
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var contactReservationSchema = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"contact_state_id": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"leafspace_status": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"status": {
+		Type:     schema.TypeString,
+		Optional: true,
```

Comment:
it's not optional from the docs: https://docs.leanspace.io/docs/api/branches/dev/leafspace/operations/create-a-contact-reservation-status-mapping
```suggestion
		Computed: true,
```